packages/fe/README.md

```diff
@@ -0,0 +1,32 @@
+[![npm (scoped)](https://img.shields.io/npm/v/@ui5-language-assistant/context.svg)](https://www.npmjs.com/package/@ui5-language-assistant/fe)
+
+# @ui5-language-assistant/fe
+
+UI5 language assistant extension with LSP for annotation-relevant building blocks
+
+## Installation
```

Comment:
Once the modules become private ones, this section isn't needed anymore.

Reply:
It is not private module